moonscript/cmd/moonc.moon

```diff
@@ -70,6 +70,13 @@ compile_file_text = (text, opts={}) ->
   parse_time = if opts.benchmark
     assert gettime!
 
+  if opts.fname and string.sub(opts.fname,-8)==".litmoon"
```

Comment:
well, since the duped code is pretty short and there was some duplication anywhere in this file, I didn't really care about it, but you're right, refactoring it would be nicer. I'll take another look if maybe `moonc` could maybe just `require "moonscript.base"` here…